utils/convertors/convertors.go

```diff
@@ -54,6 +54,8 @@ func StrToBytes(memory string) (uint64, error) {
 		"m":  0.001,
```

Comment:
@BupycHuk What is it stands for? milibytes?

Reply:
😄 looks like we were trying to reuse the same function for CPU and memory and then when we split them forgot to remove this line.

Reply:
I left 'm' unit and added a comment for the future.